package.json

```diff
@@ -65,5 +65,6 @@
   },
   "clean-publish": {
     "cleanDocs": true
-  }
+  },
+  "packageManager": "pnpm@9.9.0+sha512.60c18acd138bff695d339be6ad13f7e936eea6745660d4cc4a776d5247c540d0edee1a563695c183a66eb917ef88f2b4feb1fc25f32a7adcadc7aaf3438e99c1"
```

Comment:
Please, remove `packageManager`. It is a cool feature, but I have 50+ projects and did not find an automation tool to update `pnpm` is all of them.

Reply:
My bad! I usually omit it in my commits, but it slipped in this time.